### ISSUE TITLE
sendStartDirect Error: Request failed with status code 405

### ISSUE THREAD

**Issue by jorhett** [tonesto7/homebridge-hubitat-tonesto7#25]
## Verify the following before opening an trouble issue

**Go over all the following points, and put an `x` in all the boxes that apply.
If you're unsure about any of these, don't hesitate to ask. We're here to help!**

-  [X] That _OAuth_ is Enabled for the SmartApp under the IDE.
-  [X] The SmartApp and Device Handler are using the latest code available.
-  [X] That Both the SmartApps and Device Handlers have been _Published for You_ in the IDE.

---
## About Your Setup
-  How many devices are detected?: 12
-  Mobile App Version(Not required):
-  SmartApp Version: 2.0.2
-  Device Handler Version:
-  Homebridge Version:
-  NodeJS Version:

## Expected Behavior
**Tell us what you think should be happening**

It should connect with working IPs.

## Current Behavior
**What happens instead of the expected behavior?**

In Homebridge logs I see: 

```
[11/7/2020, 4:18:06 am] [Hubitat-v2] INFO: Sending StartDirect Request to Hubitat | LocalCommand: (true)
[11/7/2020, 4:18:06 am] [Hubitat-v2] INFO: Direct Connect Active | Listening at 172.31.2.128:8000
[11/7/2020, 4:18:06 am] [Hubitat-v2] ERROR: sendStartDirect Error: [object Object] | Message: Request failed with status code 405
[11/7/2020, 4:18:08 am] [Hubitat-v2] INFO: INFO: Your plugin version is up-to-date
[11/7/2020, 4:18:08 am] [Hubitat-v2] NOTICE: Sending Plugin Status to Hubitat | UpdateAvailable: false
[11/7/2020, 4:18:08 am] [Hubitat-v2] ERROR: sendUpdateStatus Error: [object Object] | Message: Request failed with status code 405
```

In Hubitat logs I see:
```
app:7122020-11-07 04:18:35.085 am debugHomebridge (v2.0.8) | Code versions: null
app:7122020-11-07 04:16:59.889 am warnConnection refused (Connection refused)
app:7122020-11-07 04:16:59.879 am traceHomebridge (v2.0.8) | sendDeviceRefreshCmd: null:null
app:7122020-11-07 04:16:56.920 am warnConnection refused (Connection refused)
app:7122020-11-07 04:16:56.872 am traceHomebridge (v2.0.8) | updateServicePrefs: null:null
app:7122020-11-07 04:16:51.798 am debugHomebridge (v2.0.8) | -----------------------------------------------
app:7122020-11-07 04:16:51.795 am debugHomebridge (v2.0.8) | Registered (12 Devices)
```

## Steps to Reproduce (for bugs)
**Provide a link to a live example, or an unambiguous set of steps to reproduce this bug. Include code to reproduce, if relevant**

1. Install as documented
2. copy-pasta config items from Hubitat UI to Homebridge
3. Save Hubitat app config
4. Save Homebridge config

**Comment by jorhett** [comment]
Okay, it appears local networking is not working. Changing the following works around it:

```
"use_cloud": true,
```

But I definitely want the communication to be local...

**Comment by tonesto7** [comment]
If local isn't working it likely a firewall issue on whatever device is hosting the plugin.

**Comment by jorhett** [comment]
1. The homebridge device and Hubitat are both hardwired to ethernet with no router or firewall device between them. Neither one offers firewall functionality.

2. I can login to the homebridge device and send curl requests to the firewall

3. If I deliberately tweak the urls, say adding "foo" to the end of `app_url_local` or `app_id` then I get 404 errors with complaints in the Hubitat logs about the concatenated URL

```
2020-11-07 04:38:45.290 am warn Received local request for App 712foo that does not exist, path: /startDirect/172.31.2.128/8000/2.0.8 from unknown
```

All of this indicates it's not a firewall problem. 

I'm quite competent on this stuff and happy to assist.

**Comment by jorhett** [comment]
So much has changed that I can't be certain what caused the error, but I have a theory. I was seeing a problem where after recreating the access token in App Settings, it would always be null in the Generated Config. So I'd regenerate it again, and then use that value to test.

Considering what I saw in the logs, this time I opened up the app, recreated the token and then hit *Next* and *Done* and waited for the logs to show it restarting. Then I opened the app again and viewed the Generated Configuration to acquire the token for the Homebridge configuration. This seemed to have worked. It successfully started and registered the devices.
